engine/src/main/java/org/camunda/bpm/engine/impl/cmmn/handler/TimerEventListenerItemHandler.java

```diff
@@ -0,0 +1,71 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.impl.cmmn.handler;
+
+import org.camunda.bpm.engine.impl.cmmn.behavior.CmmnActivityBehavior;
+import org.camunda.bpm.engine.impl.cmmn.behavior.EventListenerActivityBehavior;
+import org.camunda.bpm.engine.impl.cmmn.behavior.TimerEventListenerActivityBehavior;
+import org.camunda.bpm.engine.impl.cmmn.model.CmmnActivity;
+import org.camunda.bpm.engine.impl.el.Expression;
+import org.camunda.bpm.engine.impl.el.ExpressionManager;
+import org.camunda.bpm.model.cmmn.instance.CmmnElement;
+import org.camunda.bpm.model.cmmn.instance.PlanItemDefinition;
+import org.camunda.bpm.model.cmmn.instance.TimerEventListener;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * @author subhro
+ */
+public class TimerEventListenerItemHandler extends ItemHandler{
```

Comment:
Instead of extending the `ItemHandler` extend the already existing `EventListenerItemHandler`